corehq/apps/data_dictionary/management/commands/populate_case_prop_groups.py

```diff
@@ -28,7 +28,7 @@ def populate_case_prop_groups(domain):
 
     for case_prop in case_props:
         group, created = CasePropertyGroup.objects.get_or_create(
-            name=case_prop.group,
+            name=case_prop.group_obj.name,
```

Comment:
nit: The `case_prop.group_obj.name` can be changed to `case_prop.group_name` since that property also returns the same.
```suggestion
            name=case_prop.group_name,
```

Reply:
Changed